src/content/advanced/ingress/configuration/index.md

```diff
@@ -352,41 +353,20 @@ In case you want to set up a general http snippet you can define it at [NGINX Co
 
 Your Giant Swarm installation comes with a default configuration for the Ingress Controller.
 
-You can override these defaults by setting your per cluster configuration in the form of a ConfigMap named `nginx-ingress-controller-user-values`.
-
-Depending on the workload cluster release version, this ConfigMap is located either in the workload cluster or in the management cluster.
+You can override these defaults by setting your per cluster configuration in the form of a ConfigMap named `nginx-ingress-controller-user-values` in the management cluster.
 
 ### Where is the user values ConfigMap
 
 Given the cluster you are trying to configure has id: `123ab`
 
-**Workload cluster release v9.0.1 and greater:**
-
-If your cluster is on workload cluster release version v9.0.1 or greater then you will find the `nginx-ingress-controller-user-values` ConfigMap on the management cluster in the `123ab` namespace:
+You will find the `nginx-ingress-controller-user-values` ConfigMap on the management cluster in the `123ab` namespace:
```

Comment:
Our oldest clusters are on 9.0.9 so the migration is complete 🎉 and there is only one place to check.